pkg/user/tx_client.go

```diff
@@ -116,6 +120,15 @@ func WithDefaultAccount(name string) Option {
 	}
 }
 
+// WithDefaultTTLNumBlocks sets a block height-based TTL for the transaction.
+// If the transaction hasn't been included in a block by the specified height,
+// it will be considered expired and can be safely resubmitted.
```

Comment:
```suggestion
// WithDefaultTTLNumBlocks sets a height-based TTL for the transaction.
// If the transaction hasn't been included in a block by the specified height,
// it will be considered expired and can be safely resigned.
```